internal/cmd/config.go

```diff
@@ -65,6 +65,7 @@ type Config struct {
 	Distribution struct {
 		EKS struct {
 			TemplateLocation      string
+			DefaultNodeVolumeSize int
```

Comment:
Note: make sure to update the order, if you change the default config file.

Reply:
I decided to move the new config entry above the old comments - I don't understand how could I miss that - so the order remained.